.github/Dockerfile

```diff
@@ -30,14 +31,15 @@ RUN apt-get clean
 RUN apt-get autoclean
 
 # Get Rust
-RUN curl https://sh.rustup.rs -sSf | bash -s -- -y --default-toolchain 1.65.0
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y --default-toolchain 1.65.0 --profile minimal
 
 COPY --from=builder /llvm15.0 /llvm15.0/
 
 ENV PATH="/llvm15.0/bin:/root/.cargo/bin:/root/.local/share/solana/install/active_release/bin:${PATH}"
 
 # Install Solana (x86-64 only, does not work on arm)
-RUN if test `arch` = x86_64; then sh -c "$(curl -sSfL https://release.solana.com/v1.11.10/install)"; fi
+RUN if test `arch` = x86_64; then sh -c "$(curl -sSfL https://release.solana.com/v1.11.10/install)"; fi && \
+	cargo install --git https://github.com/coral-xyz/anchor --tag v0.27.0 anchor-cli && \
+	rm -rf /root/.cargo/registry && \
+	find /root/.local/share/solana/install/releases/1.11.10/solana-release/bin/ -type f -executable -exec sh -c "file '{}' | grep -q 'not stripped' " \; -print | xargs strip
```

Comment:
@seanyoung I think you could update the Solana version. v1.11.0 seems a bit old. What do you think?

Reply:
Yes, don't update it to 1.15 though (probably 1.14.16 is best)

Reply:
How can I test this?

Reply:
We currently use one label for the ci image, `ci`. How about changing the label in the PR to something else (in `.github/workflows/ci-image.yml`) and then you can build a new image without breaking anything.

Then, you can create a PR the solang repo and switch the ci image.

There is another step that can make this easier. If you create this pull request on a branch in `hyperledger/solang-llvm` rather than in your fork, then you can run the `build ci image` on your branch before merging.

Hope that makes some sense.

Reply:
I'm testing as well: https://github.com/hyperledger/solang-llvm/actions/runs/4512537364

This includes both the haircut and new solana version

Reply:
I tested it, but I get

```
  1) ChildContract
       Create Contract:
     Error: failed to send transaction: Transaction simulation failed: Blockhash not found
```
There is an issue for this already. https://github.com/hyperledger/solang/issues/1180

Reply:
Do you know by chance which version introduces this regression? Otherwise, we can just leave the solana version and only merge what's in the PR now

Reply:
I don't know when the problem appears. I'd have to test each version manually. You can just merge the PR as it is.